tests/test_cli.py

```diff
@@ -287,6 +288,11 @@ def test_format_2(format, model):
     )
 
 
+@pytest.mark.skipif(
```

Comment:
It passes for MacOS but fails on Linux, because graphviz should be installed inside of container used by cibuildwheels. I leave the test only for MacOS if it is OK.

Reply:
I will try to install `graphviz` in `manylinux` container. If it works then this test should pass on Linux too.